src/App.js

```diff
@@ -1,16 +1,35 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import chatMessages from './data/messages.json';
+import ChatLog from './components/ChatLog';
+
 
 const App = () => {
+  const [totalLikes, setTotalLikes] = useState(0)
+
+  const updateLikes = (liked) => {
+    setTotalLikes((prevLikes) => prevLikes + (liked ? 1 : -1));
+  };
+
+  const addHearts = () => {
+    let hearts = 0;
+
+    for (let i = 0; i < totalLikes; i++) {
+      hearts += 1
+    }
+
+    return hearts + ' ❤️s';
+  };
```

Comment:
Since `totalLikes` is an integer, we could remove this function and use `totalLikes + ' ❤️s'` anywhere we're calling `addHearts`.